wpilibNewCommands/src/main/native/include/frc2/command/CommandScheduler.h

```diff
@@ -191,6 +192,12 @@ class CommandScheduler final : public wpi::Sendable,
   void UnregisterSubsystem(std::initializer_list<Subsystem*> subsystems);
   void UnregisterSubsystem(std::span<Subsystem* const> subsystems);
 
+  /**
+   * @brief Get all registered subsytems as an immutable vector
+   *
+   */
+  const std::vector<Subsystem*> GetAllRegisteredSubsystems();
```

Comment:
```suggestion
  std::vector<Subsystem*> GetAllRegisteredSubsystems() const;
```
Two things- First, since we're making a copy, we don't need to prevent callers from modifying the returned vector. Second, since this is a getter method that doesn't modify the scheduler, we should mark the method as const (with the const at the end) so that it could be called on const CommandScheduler instances. (In this particular case, it doesn't matter much, but it's still good practice)